populate_test_db.sql

```diff
@@ -0,0 +1,287 @@
+);
+
+CREATE TABLE IF NOT EXISTS settlement_scores
+(
+  auction_id       bigint PRIMARY KEY,
+  winner           bytea          NOT NULL,
+  winning_score    numeric(78, 0) NOT NULL,
+  reference_score  numeric(78, 0) NOT NULL,
+  block_deadline   bigint         NOT NULL,
+  simulation_block bigint         NOT NULL
+);
+
+-- Populated after block finalization via transactionReceipt.
+CREATE TABLE IF NOT EXISTS settlement_observations
+(
+  block_number        bigint         NOT NULL,
+  log_index           bigint         NOT NULL,
+  gas_used            numeric(78, 0) NOT NULL,
+  effective_gas_price numeric(78, 0) NOT NULL,
+  surplus             numeric(78, 0) NOT NULL,
+  fee                 numeric(78, 0) NOT NULL,
+
+  PRIMARY KEY (block_number, log_index)
+);
+
+CREATE TABLE IF NOT EXISTS auction_prices
+(
+  auction_id bigint NOT NULL,
+  token bytea NOT NULL,
+  price  numeric(78, 0) NOT NULL,
+
+  PRIMARY KEY (auction_id, token)
+);
+
+-- orders table
+CREATE TYPE OrderKind AS ENUM ('buy', 'sell');
+CREATE TYPE SigningScheme AS ENUM ('presign', 'eip712', 'ethsign');
+CREATE TYPE TokenBalance AS ENUM ('erc20', 'external');
+CREATE TYPE OrderClass AS ENUM ('market', 'limit');
+
+CREATE TABLE orders (
+    uid bytea PRIMARY KEY,
+    owner bytea NOT NULL,
+    creation_timestamp timestamptz NOT NULL,
+    sell_token bytea NOT NULL,
+    buy_token bytea NOT NULL,
+    sell_amount numeric(78,0) NOT NULL,
+    buy_amount numeric(78,0) NOT NULL,
+    valid_to bigint NOT NULL,
+    fee_amount numeric(78,0) NOT NULL,
+    kind OrderKind NOT NULL,
+    partially_fillable boolean NOT NULL,
+    signature bytea NOT NULL,
+    cancellation_timestamp timestamptz,
+    receiver bytea NOT NULL,
+    app_data bytea NOT NULL,
+    signing_scheme SigningScheme NOT NULL,
+    settlement_contract bytea NOT NULL,
+    sell_token_balance TokenBalance NOT NULL,
+    buy_token_balance TokenBalance NOT NULL,
+    full_fee_amount numeric(78,0) NOT NULL,
+    class OrderClass NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS order_quotes
+(
+  order_uid bytea PRIMARY KEY,
+  gas_amount double precision NOT NULL,
+  gas_price double precision NOT NULL,
+  sell_token_price double precision NOT NULL,
+  sell_amount numeric(78, 0) NOT NULL,
+  buy_amount numeric(78, 0) NOT NULL,
+  solver bytea NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS trades
+(
+  block_number bigint         NOT NULL,
+  log_index    bigint         NOT NULL,
+  order_uid    bytea          NOT NULL,
+  sell_amount  numeric(78, 0) NOT NULL,
+  buy_amount   numeric(78, 0) NOT NULL,
+  fee_amount   numeric(78, 0) NOT NULL,
+
+  PRIMARY KEY (block_number, log_index)
+);
+
+CREATE TABLE IF NOT EXISTS order_executions
+(
+  order_uid bytea NOT NULL,
+  auction_id bigint NOT NULL,
+  reward double precision NOT NULL,
+  surplus_fee numeric(78, 0) NOT NULL,
+  solver_fee numeric(78, 0),
+
+  PRIMARY KEY (order_uid, auction_id)
+);
+
+CREATE TYPE PolicyKind AS ENUM ('priceimprovement', 'volume');
```

Comment:
The naming in the database has changed:
```suggestion
CREATE TYPE PolicyKind AS ENUM ('surplus', 'volume');
```